File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include <ctype.h>
#include <limits.h>

#define TRUE 1 
#define FALSE 0


#define ROW 20000
#define COL 100

int main(){
    int N;
    scanf("%d", &N);

    int num = 666;
    int count = 0;
    while (TRUE) {
        char str[20];
        sprintf(str, "%d", num);
        
        if (strstr(str, "666") != NULL) {
            count++;
            if (count == N) {
                printf("%d\n", num);
                break;
            }
        }
        
        num++;
    }

    return 0;
}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.py
import math
import sys
input = sys.stdin.readline


# 666 들어간 종말 수 찾기 


if __name__ == "__main__":
    N = int(input())
    if N == 1:
        print("666")
    elif N == 2:
        print("1666")
    else:
        count = 2
        num = 2665

        while True:
            num += 1
            if "666" in str(num):
                count += 1
                if count == N:
                    print(num)
                    break
